Jenkinsfile

```diff
@@ -917,16 +931,55 @@ pipeline {
                                             'spdk-devel libfabric-devel '+
                                             'pmix numactl-devel ' +
                                             'libipmctl-devel ' +
-                                            'python36-tabulate' +
+                                            'python36-tabulate ' +
                                             qb_inst_rpms
-
                     }
                     post {
                       always {
-                            unitTestPost()
+                            unitTestPost valgrind_stash: 'centos7-gcc-unit-valg'
                         }
                     }
                 }
+                stage('Unit test Bullseye') {
+                    when {
+                      beforeAgent true
+                      expression { ! skip_stage('bullseye', true) }
+                    }
+                    agent {
+                        label 'ci_vm1'
+                    }
+                    steps {
+                        script {
+                            if (quickbuild()) {
+                                // TODO: these should be gotten from the Requires: of RPMs
+                                qb_inst_rpms = " spdk-tools mercury boost-devel"
+                            }
+                        }
+                        unitTest timeout_time: 60,
+                                 ignore_failure: true,
+                                 inst_repos: daos_repos(),
+                                 inst_rpms: 'gotestsum openmpi3 ' +
+                                            'hwloc-devel argobots ' +
+                                            'fuse3-libs fuse3 ' +
+                                            'boost-devel ' +
+                                            'libisa-l-devel libpmem ' +
+                                            'libpmemobj protobuf-c ' +
+                                            'spdk-devel libfabric-devel '+
+                                            'pmix numactl-devel ' +
+                                            'libipmctl-devel ' +
+                                            'python36-tabulate' +
+                                            qb_inst_rpms
```

Comment:
Given that this is a duplicate of the existing unit test stage, can we DRY out this list somehow?

Reply:
Was https://github.com/daos-stack/daos/blob/c267ebe83825f192dadb4eb49e7142cf8f10be2d/Jenkinsfile#L119-L140 supposed to be used here?  It seems it was defined but not used to address this thread.